docs/src/content/docs/id/guides/sidebar.mdx

```diff
@@ -343,16 +396,16 @@ Tautan juga dapat menyertakan properti `attrs` untuk menambahkan atribut HTML ku
 
 Pada contoh berikut, `attrs` digunakan untuk menambahkan atribut `target="_blank"`, sehingga tautan dibuka di tab baru, dan untuk menerapkan atribut `style` kustom untuk memberi style miring pada label tautan:
 
-```js
+```js {10}
 starlight({
 	sidebar: [
 		{
-			label: 'Panduan',
+			label: 'Resources',
```

Comment:
This label can be safely translated here and the example below.

Reply:
I see, will update soon.